.github/workflows/build.yml

```diff
@@ -18,15 +18,27 @@ jobs:
 
     steps:
       - uses: actions/checkout@v2
+        with:
+          path: py42
       - name: Setup Python
         uses: actions/setup-python@v1
         with:
           python-version: ${{ matrix.python }}
       - name: Install tox
         run: pip install tox==3.17.1
       - name: Run Unit tests
-        run: tox -e py  # Run tox using the version of Python in `PATH`
+        run: cd py42; tox -e py  # Run tox using the version of Python in `PATH`
       - name: Submit coverage report
         uses: codecov/codecov-action@v1.0.7
         with:
-          file: ./coverage.xml
+          file: py42/coverage.xml
+      - name: Checkout mock server
+        uses: actions/checkout@v2
+        with:
+          ref: feature/make-containers-use-host
```

Comment:
make sure to change this line after the `code42-mock-servers` PR merges and before merging this PR